connector/connect/common/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -122,8 +122,8 @@ message Read {
   }
 
   message DataSource {
-    // (Required) Supported formats include: parquet, orc, text, json, parquet, csv, avro.
-    string format = 1;
+    // (Optional) Supported formats include: parquet, orc, text, json, parquet, csv, avro.
+    optional string format = 1;
```

Comment:
Nit: can you improve the documentation here to see what is the default format and/or how to configure it if there is a way?